DependencyInjection/Compiler/ExceptionWrapperHandlerPass.php

```diff
@@ -27,7 +28,7 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        if (interface_exists('JMS\Serializer\Handler\SubscribingHandlerInterface')) {
+        if (interface_exists(SubscribingHandlerInterface::class)) {
```

Comment:
yeah :/ we'll see if the next merges are complicated and if they are we can still revert this changes.